modules/nodes-nodes-rebooting-gracefully.adoc

```diff
@@ -85,13 +97,52 @@ $ ssh core@<master-node>.<cluster_name>.<base_domain>
 $ sudo systemctl reboot
 ----
 ====
+endif::nodes[]
+ifdef::windows[]
+. SSH into the Windows node and enter PowerShell by using the following command:
++
+[source,terminal]
+----
+C:\> powershell
+----
+
+. Restart the node by running the following command:
++
+[source,terminal]
+----
+C:\>  Restart-Computer -Force
+----
+
+. Windows nodes on Amazon Web Services (AWS) do not return to `READY` state after a graceful reboot due to an inconsistency with the EC2 instance metadata routes and the Host Network Service (HNS) networks.
++
+After the reboot, SSH into any Windows node on AWS and add the route by running the following command in a shell prompt:
++
+[source,terminal]
+----
+C:\> route add 169.254.169.254 mask 255.255.255.0 <gateway_ip>
+----
++
+where:
```

Comment:
```suggestion
In the above command:
```
Maybe add something of that sort rather than just the word "where".

Reply:
@jneczypor This is [standard formatting for additional information on what a line of a code block represents](additional information on what a line of a code block represents).